trace-utils/src/send_data.rs

```diff
@@ -193,14 +278,12 @@ impl SendData {
 
             let mut futures = FuturesUnordered::new();
             for tracer_payload in self.tracer_payloads.into_iter() {
+                let chunks = u64::try_from(tracer_payload.chunks.len()).unwrap();
                 let mut builder = HttpRequestBuilder::new()
                     .method(template.method.clone())
                     .uri(template.uri.clone())
                     .version(template.version)
-                    .header(
-                        "X-Datadog-Trace-Count",
-                        tracer_payload.chunks.len().to_string(),
-                    );
+                    .header("X-Datadog-Trace-Count", chunks.to_string());
```

Comment:
nitpicking - Ignore for this PR, but I'm not a big fan of inlined string. I believe that it's better to define constants somewhere. It's also easier to document what they are meant for.

Reply:
Since I needed to address some comments I've done this too. Besides it would've taken me more effort to just create the ticket than fixing it.